documentation/staging/content/userguide/istio/istio.md

```diff
@@ -142,6 +142,39 @@ If the WebLogic administration port is enabled on the Administration Server:
 Additionally, when Istio support is enabled for a domain, the operator
 ensures that the Istio sidecar is not injected into the introspector job's pods.
 
+#### Support for network changes in Istio v1.10 and later
+
+Prior to Istio release 1.10, the Istio Envoy proxy redirected all inbound traffic to the localhost
```

Comment:
I don't think the "prior to" paragraph is necessary and you can just begin with "Starting with Istio 1.10". Also, let's link to that article describing the change.

Reply:
I see that you did link the article later. Move the link up here :)

Reply:
Changed.  Moved the link up.

Reply:
Also, removed the text describing the previous Istio network behavior.